### DISCUSSION TITLE
EE16A&B: Designing Information Devices and Systems I&II - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#160] [Announcements]
# EE16A&B: Designing Information Devices and Systems I&II - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%94%B5%E5%AD%90%E5%9F%BA%E7%A1%80/EE16/

**Comment by QingruiSun** [comment]
所有的lab好像都需要设备，这个远程在线版是什么意思

**Reply by AdsoMelk2004** [reply]
大概是在线仿真吧

**Comment by Chenmingwang** [comment]
## Great!
### MIT 的 Introduction To EECS  也不错。
https://ocw.mit.edu/courses/6-02-introduction-to-eecs-ii-digital-communication-systems-fall-2012/

**Comment by MaryBioinfo** [comment]
a

**Comment by potatochick2020** [comment]
請問需要數學基礎嗎?

**Reply by sarendis56** [reply]
不需要的

**Reply by Yie3** [reply]
っk

**Comment by Glasscats** [comment]
可以补充一下Razavi大佬的模电1和2，基础电路

**Reply by cailikelog** [reply]
https://www.youtube.com/watch?v=yQDfVJzEymI
这是我找到的视频，不知道有无人找到合适的配套练习

**Comment by Philips-Echo** [comment]
在B站搜索的关键词是什么啊

**Reply by zhangeecs** [reply]
EECS16A

**Comment by rainstorm666666** [comment]
Homework Prob 有答案吗

**Comment by dghtucs** [comment]
建了一个Q群，一起学习讨论请入 766721253

**Comment by rainstorm666666** [comment]
似乎群号码不存在？你值得被信任吗【狗头】【狗头】


| |
tong881156700
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年5月12日 19:23 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [PKUFlyingPig/cs-self-learning] EE16A&B: Designing Information Devices and Systems I&II - CS自学指南 (Discussion #160) |

建了一个Q群，一起学习讨论请入 766721253

**Comment by Emily1412** [comment]
这个课就是数电吗

**Reply by sarendis56** [reply]
模电

**Comment by Eric30615** [comment]
有没有EE方向的学习指南呀

**Reply by HarrisYang02** [reply]
同蹲一个

**Comment by ghost** [comment]
如果学16b看的是几几年的，我看大佬只写了16a

**Reply by ghost** [reply]
我找到了2020fall

**Comment by mendax1234** [comment]
这个主要是数电还是模电啊

**Comment by mendax1234** [comment]
2020年16A的视频链接都失效了

**Comment by xikus** [comment]
ee16网站要登录了，求ee16a资源备份，如果是没有写过的原始资源就最好了

**Comment by ghost** [comment]
现在看不了了有没有其他推荐的啊

**Reply by ytoso** [reply]
现在可以访问https://eecs16a.org/

**Comment by leiedwin** [comment]
请问看完电子基础这门课能掌握传感器的使用吗？

**Comment by LaoZhaoyi** [comment]
要登陆是怎么回事捏~~~~

**Reply by Sora404zx** [reply]
https://eecs16a.org/
用这个

**Reply by revlogi** [reply]
这个感觉难度大很多啊

**Comment by ZeXaZ123** [comment]
请问资源里面是没有homework吗？

**Comment by revlogi** [comment]
这门课好像大改了，现在的要难好多 https://eecs16a.org
